lib/hammer_plug.ex

```diff
@@ -210,8 +232,20 @@ defmodule Hammer.Plug do
       {:deny, _n} ->
         on_deny_handler.(conn, [])
 
-      {:error, _reason} ->
-        on_deny_handler.(conn, [])
+      {:error, reason} ->
```

Comment:
What do you think about adding a `rescue` clause to this function? For those using Redis backend, [this](https://github.com/ExHammer/hammer-backend-redis/blob/master/lib/hammer_backend_redis.ex#L199) can happen and it's not being handled.